doc/assets/js/theme.js

```diff
@@ -26,6 +26,18 @@ function addNavBarBurgers(){
   }
 }
 
+var anchorTable = {}
```

Comment:
You should avoid vars in the global scope. You could organize all tab-related functionality into an `initializeTabs()` function and run that on page load.